wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -576,6 +578,8 @@ private void initLogging() {
 
         final LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
 
+        final LogMarkerFilter markerFilter = new LogMarkerFilter();
+
```

Comment:
This will ignore all log entries that are marked because nothing is supplied to the constructor.  Later we may enable some markers in debug builds.